portfolio.html

```diff
@@ -59,5 +60,178 @@ <h6 class="connect">LET'S CONNECT</h6>
         </ul>
       </div>
     </section>
+  </section>
+
+    <!--Portfolio "Works" section -->
+    <!-- Secound page  -->
+
+    <div class="mainContainer">
+      <section class="snapshoot">
+        <img src="img/Snapshoot Portfolio2.png" alt="Snapshoot_portfolio" />
+      </section>
+      <section class="info">
+        <h2 class="name">Multy-Post Stories</h2>
+        <ul class="info_job">
+          <li class="job">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </section>
+      <section class="decription">
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
```

Comment:
I suggest you put all the `<div class="mainContainer">` in one container and use a `section `tag on them with `display :grid` to align all the cards and then use display flex for all the individual containers. This was part of the project requirements. This means that for each `<div class="mainContainer">` that you have, will be styled using `display:flex` and the overall container will be a `<section>` that has `display:grid`